Lets Do This/Categories/NSError+LDT.m

```diff
@@ -48,6 +48,9 @@ - (NSString *)readableStringAsTitle:(BOOL)isTitle {
         NSError *error = self;
         NSDictionary *responseDict = [NSJSONSerialization JSONObjectWithData:errorData options:kNilOptions error:&error];
         NSInteger code = [responseDict valueForKeyAsInt:@"code"];
+        // We currently get an array called "errors" back on a 422: Unprocessable entity
+        // which is why this works here but not in -(NSInteger)networkResponseCode below
+        // Per https://github.com/DoSomething/northstar/pull/305 all error objects will contain a code property, so we can use -(NSInteger)networkResponseCode for any request
         NSDictionary *errorDict = responseDict[@"errors"];
```

Comment:
will need to change this if we merge https://github.com/DoSomething/northstar/pull/305